examples/basics/visualising_open_industrial_data.ipynb

```diff
@@ -0,0 +1,1104 @@
+      "Looking in indexes: https://pypi.python.org/simple, https://cognite.jfrog.io/cognite/api/pypi/snakepit/simple\n",
+      "Requirement already up-to-date: numpy in /Users/rebeccawiborgseyfarth/miniconda3/lib/python3.7/site-packages (1.16.4)\n"
+     ]
+    }
+   ],
+   "source": [
+    "!pip install cognite-sdk\n",
+    "!pip install --upgrade numpy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "CN5J9FmPBRLk"
+   },
+   "source": [
+    "#### Import the required packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "uFkQ73a44wmU"
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "import os\n",
+    "from datetime import datetime, timedelta\n",
+    "from datetime import datetime\n",
+    "from getpass import getpass\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "from sklearn.ensemble import RandomForestRegressor\n",
+    "from sklearn.linear_model import LinearRegression\n",
+    "from sklearn.metrics import mean_squared_error\n",
+    "from sklearn.metrics import r2_score\n",
+    "\n",
+    "from cognite.client import CogniteClient"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Pr0CQECZBWW3"
+   },
+   "source": [
+    "#### Connect to the Cognite Data Fusion\n",
+    "* This client object is how all queries will be sent to the Cognite API to retrieve data.\n",
+    "\n",
+    "When prompted for your API key, use the key generated by open industrial data as mentioned in the Getting Started steps."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "E8CdoRFu9n2t",
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Open Industrial Data API-KEY: ········\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/local/lib/python3.7/site-packages/cognite/client/_cognite_client.py:57: UserWarning: You are using version 1.0.0a35 of the SDK, however version 1.0.0b1 is available. Upgrade or set the environment variable 'COGNITE_DISABLE_PYPI_VERSION_CHECK' to suppress this warning.\n",
+      "  debug=debug,\n",
+      "/usr/local/lib/python3.7/site-packages/ipykernel_launcher.py:1: UserWarning: Authenticated towards inferred project 'publicdata'. Pass project to the CogniteClient constructor or set the environment variable 'COGNITE_PROJECT' to suppress this warning.\n",
+      "  \"\"\"Entry point for launching an IPython kernel.\n"
+     ]
+    }
+   ],
+   "source": [
+    "client = CogniteClient(api_key=getpass(\"Open Industrial Data API-KEY: \"), client_name=\"OID_example\")"
```

Comment:
Add project="publicdata" here to suppress the inferred project warning?

Reply:
Done